include/zephyr/bluetooth/bluetooth.h

```diff
@@ -517,6 +525,60 @@ size_t bt_data_get_len(const struct bt_data data[], size_t data_count);
  */
 size_t bt_data_serialize(const struct bt_data *input, uint8_t *output);
 
+struct bt_le_local_features {
```

Comment:
Do we want to add Filter accept list size to this struct?

Reply:
I think that would make sense, but the Zephyr host isn't reading that at the moment, so that would be a new feature